syntax = "proto3";

service OrderService {
    rpc CalculateTotal (OrderRequest) returns (OrderResponse);
}

message OrderRequest {
    string productId = 1;
    int32 quantity = 2;
}

message OrderResponse {
    double total = 1;
}